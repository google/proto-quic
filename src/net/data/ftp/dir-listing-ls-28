drwxr-x   2       10     4096 Dec 10 14:32 pollq
drwxr-x   4        0     4096 Jul 28 01:44 etc
drwxrwx   2       10     4096 Jul 28 02:41 tmp
drwxr-x   2       10     4096 Jul 28 02:00 status
drwxr-x   2        0     4096 Jul 27 23:21 bin
