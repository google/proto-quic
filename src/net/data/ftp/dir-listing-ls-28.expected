d
pollq
-1
1993
12
10
14
32

d
etc
-1
1994
7
28
1
44

d
tmp
-1
1994
7
28
2
41

d
status
-1
1994
7
28
2
0

d
bin
-1
1994
7
27
23
21
